### PR TITLE
Add easy auth

### PR DESCRIPTION
**Pull request by FrimJo**
Add support to setup multiple easy auth providers when using the template

### CODE REVIEW

File: remix.init/index.js [hunk 1]
```diff
@@ -66,6 +68,8 @@ async function main({ rootDirectory }) {
     newEnv += `SHADOW_DATABASE_URL="${answers.shadowConnectionString}${EOL}"`;
   }
 
+  await setupEasyAuth({ subscriptionId: '81a77569-b654-4cae-8282-980ac9136597', location: 'westus2', resourceGroup: 'ExampleGroup3', appName: 'mychiptunestackappsite', url: 'https://mychiptunestackappsite.azurewebsites.net' })
```
Comment (maxpaj) [inline remix.init/index.js]: Can we read these variables from the terminal or through `az` CLI?

**Comment by FrimJo** [comment]
Will update this with support for only google or none as provider

File: remix.init/setup-easy-auth.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+  return async function (provider, clientId, clientSecret) {
+    await execSync(`az webapp auth ${provider} update --name ${appName} --client-id ${clientId} --client-secret ${clientSecret} --yes`)
+  }
+}
+
+module.exports = setupEasyAuth
+
+
+// az webapp auth update --resource-group chiptunestac9c98 --name chiptunestac9c98
```
Comment (maxpaj) [inline remix.init/setup-easy-auth.js]: Should we remove this commented code? In general, I like to keep commits free of commented code

File: remix.init/index.js (hunk 1 above)
Comment (FrimJo) [reply remix.init/index.js]: The data for the varaibles which can be passed here will be received earlier in the script after spinning up the environemt.

File: remix.init/setup-easy-auth.js (hunk 2 above)
Comment (FrimJo) [reply remix.init/setup-easy-auth.js]: fixed in [remove comments](https://github.com/FrimJo/chiptune-stack/pull/1/commits/ccb9a3a5ad6d9b6144c6010f56aebc7e16397e0a).

**Pull request merged**